app/controllers/webhooks/pull_request_updates_controller.rb

```diff
@@ -0,0 +1,54 @@
+module Webhooks
+  class PullRequestUpdatesController < BaseController
+    def create
+      ::Webhooks::ProcessPullRequestUpdate.(action, github_username,
+        { url: url,
+          html_url: html_url,
+          labels: labels,
+          state: state,
+          pr_id: pr_id,
+          repo: repo,
+          merged: merged? })
```

Comment:
```suggestion
      ::Webhooks::ProcessPullRequestUpdate.(action, github_username, {
          url: url,
          html_url: html_url,
          labels: labels,
          state: state,
          pr_id: pr_id,
          repo: repo,
          merged: merged? 
        })
```

Reply:
I thought that was what I had before, but I guess I did not add the field: prefixes. Will fix!